trail/query/views.py

```diff
@@ -1,6 +1,42 @@
-from django.http import HttpResponse
+from django import forms
+# from django.forms import ModelForm
+from django.apps import apps
+from django.shortcuts import render
+
+
+Metadata = apps.get_model('upload', 'Metadata')
+# Needs to think about adding SELECT DISTINCT etc.
+
+
+class MetadataForm(forms.Form):
+    unique_instrument = ["DECam", "ff09", "text", "option"]
```

Comment:
Please populate this list with actual instrument names that exist in the metadata database. I know for a fact "text" and "option" are invalid 🙂

Reply:
If you can query for unique values from `Metadata` instrument column that would be even better.

Reply:
Fixed to instrument2 and instrument3, but I need more information about what instruments can be there.

Reply:
@DinoBektesevic I agree this would be better, but we swapped in a hard-coded value for SDSS for now.

Reply:
`Metadata.objects.values('instrument').distinct()`

is the invocation. This returns a `QuerySet` (a list) of dicts of key instrument and instrument names I think.